integration/test/app/kubeconfig/kubeconfig_test.go

```diff
@@ -19,17 +19,17 @@ import (
 
 	"github.com/giantswarm/app-operator/integration/key"
 	"github.com/giantswarm/app-operator/integration/templates"
+	"github.com/giantswarm/app-operator/pkg/label"
 )
 
 const (
-	namespace            = "giantswarm"
-	chartOperatorVersion = "chart-operator.giantswarm.io/version"
-	targetNamespace      = "test"
-	testAppCatalogName   = "test-app-catalog"
+	clusterName = "kind-kind"
+	namespace   = "giantswarm"
 )
 
-// TestAppLifecycleUsingKubeconfig perform same tests as TestAppLifeCycle except it using kubeConfig spec
-func TestAppLifecycleUsingKubeconfig(t *testing.T) {
+// TestAppWithKubeconfig checks app-operator can bootstrap chart-operator
+// when a kubeconfig is provided.
```

Comment:
I'd like to simplify the kubeconfig test. Currently it repeats all the tests in the basic test which makes it very long.

With this it now just checks that the bootstrap resources work correctly.